package/src/components/AudioPlayer/Interface/Controller/Input/Progress/WaveformProgress.css

```
@@ -0,0 +1,20 @@
+.waveform-wrapper[data-active="false"] {
+  height: 0;
+  opacity: 0;
+  pointer-events: none;
+}
+
+.waveform-wrapper #rm-waveform {
+  width: 100%;
+  overflow: hidden;
+}
+
+.waveform-wrapper #rm-waveform wave {
```

Stylelint (17.5.0)

[error] Unexpected unknown type selector "wave" (selector-type-no-unknown)

(selector-type-no-unknown)